### DISCUSSION TITLE
How to add a custom site-package folder for the app usage?

### DISCUSSION THREAD

**Discussion by zerocewl** [pyinstaller/pyinstaller#6862] [Help]
Is it possible to add a custom `site-package` folder (path) to the pyinstaller config which can be used during the execution of the app?

The packages inside this folder should be available in the python env used by the app.

E.g. the user want some third party python packages installed (which are not bundled with the app), but which should survive any re installation.

(Sry if this is a duplicate question, but i haven't found anything in the docs / issues.)

**Comment by rokm** [comment]
External library directories can be added by modifying `sys.path`, same as in regular python.
